CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Features
 
+- Sentry Swift framework (#2162)
```

Comment:
`m`: I think this PR fixes https://github.com/getsentry/sentry-cocoa/issues/1980, which is the only new feature in this PR. Therefore I would adapt the name to reflect this change.